src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -289,6 +297,12 @@ internal bool HasAllInformation
             set => ChangeProjectProperty(ref _hasAllInformation, value, s => s.WithHasAllInformation(Id, value));
         }
 
+        internal bool IsPrimary
```

Comment:
Any tests for this? There's already a harness set up in https://github.com/dotnet/roslyn/tree/master/src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioProjectTests so just add another file and maybe add something small?

Reply:
sure.